build.gradle

```diff
@@ -89,11 +89,13 @@ allprojects {
                 classpath += files(project(':checker').tasks.getByName('shadowJar').archivePath)
                 options.taglets 'org.checkerframework.taglet.ManualTaglet'
                 options.tagletPath(project(':util').sourceSets.taglet.output as File[])
-                options.links = ['https://docs.oracle.com/javase/8/docs/api/', 'https://docs.oracle.com/javase/8/docs/jdk/api/javac/tree/']
+                options.links = ['https://docs.oracle.com/javase/8/docs/api/', 'https://docs.oracle.com/javase/8/docs/jdk/api/javac/tree/', 'https://checkerframework.org/api/']
                 // This file is looked for by Javadoc.
                 file("${destinationDir}/resources/fonts/").mkdirs()
                 ant.touch(file: "${destinationDir}/resources/fonts/dejavu.css")
             }
+            // Turn javadoc warnings into errors.
+            options.addStringOption('Xwerror', '-quiet')
```

Comment:
This causes the `javadoc` tasks to fails.  (It does not cause `allJavadoc` to fail, which Travis uses, because that task does not use the `javadoc` tasks.)